### PR TITLE
Create component reason for change

### PR DESCRIPTION
**Pull request by diegoAndres01**
Create component reason for change
Card Jira
https://inube.atlassian.net/browse/CB-135

### CODE REVIEW

**Review by carin91 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -47,7 +47,6 @@
     "@storybook/addon-essentials": "^8.2.3",
```
Comment (carin91) [inline package.json:47]: Avoid large changes on package-lock.json

File: src/components/ReasonForChange/stories/ReasonForChange.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { parameters, props } from "./props";
+import { ReasonForChange, ReasonForChangeProps } from "..";
+
+const meta: Meta<typeof ReasonForChange> = {
+  title: "components/ReasonForChange",
```
Comment (carin91) [inline src/components/ReasonForChange/stories/ReasonForChange.stories.tsx:7]: Adjust the location of the story.
image
by including: title: "components/inputs/ReasonForChange"

File: src/components/ReasonForChange/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <Stack margin="10px 0px">
+                <Textarea
+                    id="reasonChange"
+                    label={label}
+                    onChange={handleChange}
+                    placeholder={placeholder}
+                    required={required}
+                    value={valueTextarea}
+                    maxLength={120}
+                ></Textarea>
```
Comment (carin91) [inline src/components/ReasonForChange/index.tsx:37]: Could you avoid the 2 tags and use a self closing tag? considering that the Textarea component does not have children

File: src/components/ReasonForChange/index.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    const [valueTextarea, setTextarea] = useState(value);
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setTextarea(event.target.value);
+        onHandleChange(event);
+    };
+
+    return (
+        <Stack direction="column" margin="10px 5px">
+            <Text size="large" appearance="gray" weight="bold" margin="10px 0px">Cambio</Text>
```
Comment (carin91) [inline src/components/ReasonForChange/index.tsx:27]: Avoid magical texts, register them on a config file.

File: src/components/ReasonForChange/index.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+import { Stack } from "@inubekit/stack";
+import { Text } from "@inubekit/text";
+import { Textarea } from "@inubekit/textarea";
+
+interface ReasonForChangeProps {
+    label: string;
+    onHandleChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    placeholder: string;
+    required?: boolean;
```
Comment (carin91) [inline src/components/ReasonForChange/index.tsx:11]: There is a case of use when this ReasonForChange is not required?, i would not include this optional prop

**Review by Brandonz07 — APPROVED** [review]

**Review by Brandonz07 — APPROVED** [review]

**Pull request merged**